src/renderer/logic/io/FileIO.ts

```diff
@@ -0,0 +1,106 @@
+import { PathLike } from "fs";
+import { Playlist } from "../playlist";
+import { ALLOWED_AUDIO_EXTENSIONS, ALLOWED_EXTENSIONS, ALLOWED_PLAYLIST_FORMATS, AMETHYST_PLAYLIST_EXTENSION } from "@shared/constants";
+import { amethyst } from "@/amethyst";
+import fs from "fs";
```

Comment:
should not attempt to import fs from node.js, theres no fs in javascript/chromium